slash/react/src/List/ContentItemDuo/ContentItemDuo.tsx

```diff
@@ -0,0 +1,72 @@
+import checkIcon from "@material-symbols/svg-400/outlined/check.svg";
+import closeIcon from "@material-symbols/svg-400/outlined/close.svg";
+import { useMemo } from "react";
+import { Variants } from "../../Button/Button.client";
+import { getComponentClassName } from "../../Form/core";
+import { Button, Svg } from "../../client";
+
+type ContentItemDuoProps = {
+  label: string;
+  value: string;
+  isVertical?: boolean;
+  className?: string;
+  classModifier?: string;
+  isShowingDoneIcon?: boolean;
+  isShowingCloseIcon?: boolean;
+  buttonText?: string;
+  onButtonClick?: () => void;
+};
+
+export const ContentItemDuo = ({
+  label,
+  value,
+  isVertical = false,
+  className,
+  classModifier,
+  isShowingDoneIcon = false,
+  isShowingCloseIcon = false,
+  buttonText,
+  onButtonClick,
+}: ContentItemDuoProps) => {
+  const componentClassName = useMemo(() => {
+    let newClassModifier = classModifier ?? "";
+
+    if (isVertical) {
+      newClassModifier += " vertical";
+    }
+
+    return getComponentClassName(
+      className,
+      newClassModifier,
+      "af-content-item-duo",
+    );
+  }, [classModifier, className, isVertical]);
+
+  const iconContainerModifier = useMemo(
+    () =>
+      isShowingDoneIcon
+        ? " af-content-item-duo__icon--done"
+        : " af-content-item-duo__icon--close",
```

Comment:
```suggestion
      ` af-content-item-duo__icon--${isShowingDoneIcon ? "done" : "close"`,
```

Reply:
Comment ais-je pu passer à côté 😆

Reply:
ça sert à ça les reviews 🙂

Reply:
Resolved in https://github.com/AxaFrance/design-system/pull/450/commits/cdc14a88d8bcb1b2fcaaf007358c9710ac3a5f2d